src/components/App.js

```diff
@@ -0,0 +1,49 @@
+import React, { Component } from 'react';
+import { render } from 'react-dom';
+
+const ListItem = (props) => {
```

Comment:
Maybe spend some time and look at how to break this out into another file. That's sort of the react way of doing things.